package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test": "jest  --detectOpenHandles",
     "start": "tsc && node dist/server.js",
+    "migrate": "npx sequelize-cli db:migrate",
```

Comment:
This is good. Here's something else i need you to do. 
- In some cases we might need to undo migrations, so add another script to do just that. 
- We need seeds in our app. Look it up from sequelize. Create a user seed specifically for the admin along with his credentials. In the future we'll even create seeds for the product model. 
- Create a seed script that will run two other scripts where one will undo all seeds then redo them. This script should also be run in production to make sure the admin is always present.